contracts/SCO30_Andrew_LoudAndHeavy.zargabad/factions.txt

```diff
@@ -0,0 +1,5 @@
+Theseus INC PMC:
+Use INDEP Theseus INC faction
+
+Taliban:
+Use OPFOR Taliban Veterans & Taliban Militia factions
```

Comment:
@Andrew-Libby These are only needed on the templates, you don't need to worry about them for one-off missions